From 84d4d0be9051ecf599fc04ebc2cf8e052a20cf74 Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Wed, 19 Aug 2015 15:24:24 +0800
Subject: [PATCH 0475/1691] MLK-11290-02 ARM: dts: imx7d-12x12-lpddr3-arm2: add
 enet dts file to makefile

commit 84d4d0be9051ecf599fc04ebc2cf8e052a20cf74 from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

Add enet dts file to makefile.

Signed-off-by: Fugang Duan <B38611@freescale.com>
---
 arch/arm/boot/dts/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 04b77c8..94c612f 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -321,6 +321,7 @@ dtb-$(CONFIG_SOC_IMX7D) += \
 	imx7d-sdb.dtb \
 	imx7d-12x12-lpddr3-arm2-m4.dtb \
 	imx7d-12x12-lpddr3-arm2-ecspi.dtb \
+	imx7d-12x12-lpddr3-arm2-enet2.dtb \
 	imx7d-sdb-m4.dtb \
 	imx7s-colibri-eval-v3.dtb
 dtb-$(CONFIG_SOC_LS1021A) += \
-- 
1.9.1

